pkg/deployment/resources/pod_inspector.go

```diff
@@ -171,6 +181,18 @@ func (r *Resources) InspectPods(ctx context.Context, cachedStatus inspectorInter
 				log.Debug().Str("pod-name", pod.GetName()).Msg("Updating member condition Terminated to true: Pod Failed")
 				updateMemberStatusNeeded = true
 				nextInterval = nextInterval.ReduceTo(recheckSoonPodInspectorInterval)
+
+				if pod.Spec.RestartPolicy == core.RestartPolicyAlways {
```

Comment:
it seems that we have the same code here and in the line 108, maybe we should consider to create a function?